src/plugins/saved_objects_management/public/management_section/objects_table/components/relationships.tsx

```diff
@@ -32,7 +32,7 @@ import {
   EuiText,
   EuiSpacer,
 } from '@elastic/eui';
-import { FilterConfig } from '@elastic/eui/src/components/search_bar/filters/filters';
+import { SearchFilterConfig } from '@elastic/eui/src/components/search_bar/filters/filters';
```

Comment:
BTW, any plan to have this exposed from `@elastic/eui` directly, as this is a type used in public components / apis?

Reply:
We do in fact export from `@elastic/eui` now, but didn't catch this path. I've updated this to import from the top-level.